helpers.js

```diff
@@ -10,3 +10,27 @@ export const supportedTopicCategories = {
   1: { longName: "General", name: "general" },
   3: { longName: "Contributors", name: "contributors" },
 };
+
+export const formatDateDiff = (recent, old) => {
+  const timeDifference = new Date(recent) - new Date(old);
+  const seconds = timeDifference / 1000;
+  const minutes = seconds / 60;
+  const hours = minutes / 60;
+  const days = hours / 24;
+
+  if (seconds < 60) return "1m";
+  if (minutes < 60) return Math.round(minutes) + "m";
+  if (hours < 24) return Math.round(hours) + "h";
+  if (days < 30) return Math.round(days) + "d";
+  if (bumpDate.getFullYear() != new Date(Date.now()).getFullYear()) {
```

Comment:
Hi @ChefDaddyJay ,

After your last modifications, this line was removed :
```
const bumpDate = new Date(bumpedAt);
```
So the `bumpDate` variable is now undefined.